tests/MiniProfiler.Tests/App.config

```diff
@@ -34,4 +34,7 @@
       <add name="SQLite Data Provider" invariant="System.Data.SQLite" description=".NET Framework Data Provider for SQLite" type="System.Data.SQLite.SQLiteFactory, System.Data.SQLite" />
     </DbProviderFactories>
   </system.data>
+  <appSettings>
+    <add key="xunit.parallelizeTestCollections" value="false"/>
```

Comment:
I'm already doing this today with the tests that have different storages, etc. required. At a higher level: it doesn't matter if it's hard to do, it matters if it's *the right thing* to do. Disabling parallelization doesn't solve the problem, it masks a problem.